manual/cops_metrics.md

```diff
@@ -18,7 +19,7 @@ Max | `15` | Integer
 
 ### References
 
-* [http://c2.com/cgi/wiki?AbcMetric](http://c2.com/cgi/wiki?AbcMetric)
+* [http://c2.com/cgi/wiki?AbcMetric and https://en.wikipedia.org/wiki/ABC_Software_Metric](http://c2.com/cgi/wiki?AbcMetric and https://en.wikipedia.org/wiki/ABC_Software_Metric)
```

Comment:
Whoops, this isn’t a valid Markdown link.

Reply:
This is being auto-generated from the Reference in `default.yml`. As far as I can tell, it is the only cop that we have multiple references for. Should I expand our reference functionality to support multiple references? Any other thoughts on workarounds?

Reply:
It looks easy enough to modify our code to accept an array of references. My main concern is that I would want to change the config from `Reference` to `References`, and this would break any custom cops or rubocop extension projects that list use `Reference`. It is an easy enough fix, but I would hate to break something without warning.

I could also deprecate support for `Reference` in favor of `References` or maintain both. I'll start writing some code to support multiple references and we can hash out some details later.

Reply:
I could also "overload" `Reference` and allow a single element or a list of elements. I would still want to eventually change the naming to `References`.

Reply:
I have updated the code to allow multiple urls in the `Reference` section. It is still a little weird since singular `Reference` can now contain multiple elements.